### ISSUE TITLE
Cache warnings in prepRawBiomassMap tests

### ISSUE THREAD

**Issue by achubaty** [PredictiveEcology/LandR#93]
```r
══ Warnings ════════════════════════════════════════════════════════════════════
── Warning ('test-prepRawBiomassMap.R:57:3'): test prepRawBiomassMap ───────────
In do.call(what = prepInputs, args = Args): The Cache file exists, but there is no database entry for it; removing the file and rerunning the call
Backtrace:
     ▆
  1. ├─LandR::prepRawBiomassMap(...) at test-prepRawBiomassMap.R:57:2
  2. │ └─reproducible::Cache(do.call, what = prepInputs, args = Args)
  3. │   └─reproducible:::evalTheFun(...)
  4. │     ├─base::withCallingHandlers(...)
  5. │     ├─base (local) out(...)
  6. │     └─reproducible (local) `<fn>`(...)
  7. │       └─reproducible:::process(...)
  8. │         ├─base::withCallingHandlers(...)
  9. │         └─reproducible::Cache(...)
 10. │           └─reproducible:::searchInRepos(...)
 11. │             └─base::warning(...)
 12. ├─base::.signalSimpleWarning(...)
 13. │ └─base::withRestarts(...)
 14. │   └─base (local) withOneRestart(expr, restarts[[1L]])
 15. │     └─base (local) doWithOneRestart(return(expr), restart)
 16. └─reproducible (local) `<fn>`(`<smplWrnn>`)
── Warning ('test-prepRawBiomassMap.R:88:3'): test prepRawBiomassMap ───────────
In do.call(what = prepInputs, args = Args): The Cache file exists, but there is no database entry for it; removing the file and rerunning the call
Backtrace:
     ▆
  1. ├─LandR::prepRawBiomassMap(...) at test-prepRawBiomassMap.R:88:2
  2. │ └─reproducible::Cache(do.call, what = prepInputs, args = Args)
  3. │   └─reproducible:::evalTheFun(...)
  4. │     ├─base::withCallingHandlers(...)
  5. │     ├─base (local) out(...)
  6. │     └─reproducible (local) `<fn>`(...)
  7. │       └─reproducible:::process(...)
  8. │         ├─base::withCallingHandlers(...)
  9. │         └─reproducible::Cache(...)
 10. │           └─reproducible:::searchInRepos(...)
 11. │             └─base::warning(...)
 12. ├─base::.signalSimpleWarning(...)
 13. │ └─base::withRestarts(...)
 14. │   └─base (local) withOneRestart(expr, restarts[[1L]])
 15. │     └─base (local) doWithOneRestart(return(expr), restart)
 16. └─reproducible (local) `<fn>`(`<smplWrnn>`)
── Warning ('test-prepRawBiomassMap.R:101:3'): test prepRawBiomassMap ──────────
In do.call(what = prepInputs, args = Args): The Cache file exists, but there is no database entry for it; removing the file and rerunning the call
Backtrace:
     ▆
  1. ├─LandR::prepRawBiomassMap(...) at test-prepRawBiomassMap.R:101:2
  2. │ └─reproducible::Cache(do.call, what = prepInputs, args = Args)
  3. │   └─reproducible:::evalTheFun(...)
  4. │     ├─base::withCallingHandlers(...)
  5. │     ├─base (local) out(...)
  6. │     └─reproducible (local) `<fn>`(...)
  7. │       └─reproducible:::process(...)
  8. │         ├─base::withCallingHandlers(...)
  9. │         └─reproducible::Cache(...)
 10. │           └─reproducible:::searchInRepos(...)
 11. │             └─base::warning(...)
 12. ├─base::.signalSimpleWarning(...)
 13. │ └─base::withRestarts(...)
 14. │   └─base (local) withOneRestart(expr, restarts[[1L]])
 15. │     └─base (local) doWithOneRestart(return(expr), restart)
 16. └─reproducible (local) `<fn>`(`<smplWrnn>`)
```

**Comment by achubaty** [comment]
this may be an issue with `reproducible`, but I'll keep the issue here until I have to diagnose

**Comment by achubaty** [comment]
these seem to have resolved themselves with the fixes to #92
